app/Providers/TranslationServiceProvider.php

```
@@ -4,11 +4,24 @@
 namespace App\Providers;
 
+use Illuminate\Contracts\Translation\Translator;
 use Illuminate\Translation\FileLoader;
 use Illuminate\Translation\TranslationServiceProvider as BaseTranslationServiceProvider;
 
 class TranslationServiceProvider extends BaseTranslationServiceProvider
 {
+    public function register()
+    {
+        parent::register();
+        $this->app->make(Translator::class)->handleMissingKeysUsing(function ($key, $replace, $locale, $fallback) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$replace'. (undefined)

(UnusedFormalParameter)